### PR TITLE
django 3.0 to matrix; drop django 2.1

### PR DESCRIPTION
**Pull request by auvipy**
work in progress

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -15,6 +15,7 @@ notifications:
 templates:
```
Comment (auvipy) [inline .travis.yml]: is this template part really necessary?

**Comment by auvipy** [comment]
I will fix the failing tests

File: .travis.yml (hunk 1 above)
Comment (ad-m) [reply .travis.yml]: It simplify syntax, so I recommend keep it.

**Comment by ad-m** [comment]
Missing changes in envlist in tox.ini.

**Comment by ad-m** [comment]
Missing changes for TravisCI ( https://github.com/django-guardian/django-guardian/blob/devel/contrib/travis/test.sh#L12 ). Propably we can test example project on all tested Django version.

**Comment by auvipy** [comment]
i will be back soon

**Comment by auvipy** [comment]
NameError: name 'force_text' is not defined, it should be force_str?

**Comment by auvipy** [comment]
can you check the remaining two failing tests? a new release is needed for django-rest and django 3rc1 compat

**Comment by ad-m** [comment]
```
guardian/testapp/tests/test_mixins.py:30

  /home/travis/build/django-guardian/django-guardian/guardian/testapp/tests/test_mixins.py:30: PytestCollectionWarning: cannot collect test class 'TestView' because it has a __init__ constructor (from: guardian/testapp/tests/test_mixins.py)

    class TestView(PermissionRequiredMixin, RemoveDatabaseView):
```
This seems to be a problem only with the Mixin class naming so that it is not detected as a test (it is a view).

**Comment by auvipy** [comment]
OK will check

**Comment by brianmay** [comment]
Does this PR supersede #642?

**Comment by auvipy** [comment]
yes

**Comment by ad-m** [comment]
Closed by #667

**Pull request closed without merging**